### PR TITLE
How to contribute to github

### PR DESCRIPTION
**Pull request by Mangielijah**
How to setup git/github to start contributing

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+SETTING UP GIT/GITHUB
+
+How to setup git/github and your local syterm to get started with contributing to librehealthehr
```
Comment (naveen17797) [inline CONTRIBUTE.md]: system*

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	  (now your github repository is called ‘origin’)
+
+	5) copy the lh-ehr folder from the directory you cloned it to, to your server
+
+	6)Move into the lh-ehr folder on your server from the terminal:
+	  cd lh-ehr
+
+	7) Set up a connection to the main LibreEHR github respository:
+	   git remote add upstream git://github.com/LibreHealthIO/lh-ehr.git
+	   (now the main openemr github repository is called ‘upstream’)
```
Comment (teryhill) [inline CONTRIBUTE.md]: Change openemr to LibreEHR

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+SETTING UP GIT/GITHUB
+
+How to setup git/github and your local system to get started with contributing to librehealthehr
+
+Note if on Windows download and install Git Bash and use it to run the comands below
```
Comment (teryhill) [inline CONTRIBUTE.md]: commands instead of comands

**Comment by Mangielijah** [comment]
Any more changes to the PR?

**Comment by Trodrige** [comment]
You can also add a NOTE that reminds contributors to always create a branch from the updated master branch before making a new change. I think it's important.

**Comment by aethelwulffe** [comment]
@teryhill can you see about signing off on this?
or @Trodrige

**Pull request merged**